src/App.tsx

```diff
@@ -1,61 +1,178 @@
+import React, { useState, useEffect, ChangeEvent } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const getUserById = (userId: number) => {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const getUserByName = (username: string) => {
+  const foundUser = usersFromServer.find(user => user.name === username);
+
+  return foundUser || null;
+};
```

Comment:
What happened if we will have more than one user with this name? use better way to find user)